app/src/test/java/nerd/tuxmobil/fahrplan/congress/utils/FahrplanMiscTest.kt

```diff
@@ -83,26 +70,6 @@ class FahrplanMiscTest {
 
     }
 
-    @Test
-    fun getUpcomingLecturesWithEmptyList() {
-        val appRepository = mock<AppRepository> {
-            on { loadLecturesForDayIndex(anyInt(), anyBoolean()) } doReturn emptyList()
-        }
-        assertThat(FahrplanMisc.getUncanceledLectures(appRepository, AppRepository.ALL_DAYS)).isEmpty()
-    }
-
-    @Test
-    fun getUpcomingLecturesWithAllEvents() {
-        val appRepository = mock<AppRepository> {
-            on { loadLecturesForDayIndex(anyInt(), anyBoolean()) } doReturn mutableListOf(EVENT_3001, EVENT_3002)
-        }
-        val upcomingEvents = FahrplanMisc.getUncanceledLectures(appRepository, AppRepository.ALL_DAYS)
```

Comment:
`AppRepository.ALL_DAYS` can be private now. I will change this in the next review iteration.

Reply:
:heavy_check_mark: Done.